tools/gen_onnx.sh

```diff
@@ -1,14 +1,23 @@
 #!/bin/sh
+
+# This script should be executed in pytorch root folder.
+
+TEMP_DIR=tools/temp
```

Comment:
This should be added to gitignore then? But generally, it's safer to generate a random dir with `mktemp`

Reply:
Good suggestion. Will take a look.